dev/deploy-to-container/settings_local.py

```diff
@@ -60,10 +60,11 @@
 BOFREQ_PATH = '/assets/ietf-ftp/bofreq/'
 CONFLICT_REVIEW_PATH = '/assets/ietf-ftp/conflict-reviews/'
 STATUS_CHANGE_PATH = '/assets/ietf-ftp/status-changes/'
-INTERNET_DRAFT_ARCHIVE_DIR = '/assets/archive/id'
+INTERNET_DRAFT_ARCHIVE_DIR = '/assets/collection/draft-archive'
 INTERNET_ALL_DRAFTS_ARCHIVE_DIR = '/assets/archive/id'
```

Comment:
Should `INTERNET_ALL_DRAFTS_ARCHIVE_DIR` change to `/assets/ietf-ftp/internet-drafts` to match other places?

Reply:
Yes, but not as part of this PR I think. (That should be part of a full rework of the various environments idea of the structure of /assets).